src/components/LoginForm.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+
+export default class LoginForm extends Component {
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const loginData = {};
+    loginData.username = event.target.username.value.trim();
+    loginData.password = event.target.password.value.trim();
+    const validUsername = loginData.username.length > 3;
+    const validpassword = loginData.password.length > 3;
```

Comment:
Create separate service for validation,  this conditions shoud not be set in function...